go.mod

```diff
@@ -29,10 +30,10 @@ require (
 	github.com/stretchr/testify v1.7.0
 	go.buf.build/odpf/gw/odpf/proton v1.1.9
 	go.uber.org/zap v1.19.0
-	golang.org/x/text v0.3.6
-	google.golang.org/grpc v1.40.0
+	golang.org/x/text v0.3.7
+	google.golang.org/grpc v1.45.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	gorm.io/datatypes v1.0.0
 	gorm.io/driver/postgres v1.0.8
-	gorm.io/gorm v1.20.12
+	gorm.io/gorm v1.21.4
```

Comment:
Not related with this but I still see this one is importing gorm and in salt https://github.com/odpf/salt/tree/main/audit/repositories is the one that uses gorm. I think next the audit/repositories could use this db package so we can get rid of gorm dependency from salt.

Reply:
+1

Reply:
Yes, +1 on this. We can add a separate issue for that. Although it will also link to Guardiann deprecating gorm as an ORM.